apm-agent-core/src/test/java/co/elastic/apm/agent/logging/Log4j2ConfigurationFactoryTest.java

```diff
@@ -69,6 +71,8 @@ void testLogFilePlainText(@TempDir Path tempDir) {
         assertThat(appender).isInstanceOf(RollingFileAppender.class);
         assertThat(((RollingFileAppender) appender).getFileName()).isEqualTo(logFile);
         assertThat(appender.getLayout()).isInstanceOf(PatternLayout.class);
+        // workaround for [windows10]
+        configuration.getAppender("rolling").stop();
```

Comment:
Can this be done for all tests through a `@AfterEach` method? For example, have an appender field that the tests set when they starts and stop them after all tests.

Reply:
Added @AfterEach fir this two tests with @TempDir.
```
    @AfterEach
    public void stopRollingAppender() {
        if (appender != null) {
            appender.stop();
        }
    }
```